news/this-week-in-fluvio-0001.md

```diff
@@ -152,19 +146,17 @@ most errors up and out using `?`, like we do in the example above.
 
 > *and `Send + Sync + 'static`. We use the [`eyre`] crate to capture returned errors.
 
-[`eyre`]: https://github.com/yaahc/eyre
-
 This SmartStream parses incoming records as integers, then filters out odd numbers.
 When we run a consumer with this SmartStream, we can see the filtered data, and we
 can see the consumer deliver our error to us if we give an input that can't be parsed
 as an integer.
 
-<video controls width="860px" title="Producing integers to the filter-odd topic, followed by the word nine, results in an error">
+<!-- <video controls width="860px" title="Producing integers to the filter-odd topic, followed by the word nine, results in an error">
```

Comment:
Let's delete instead of comment.

Reply:
We actually have this video file with the site, I have to figure out why it doesn't serve or play

Reply:
after fiddling with this for a while, I'm deleteing the comment.  mov files don't seem to get built into the static outputs at all